test/e2e/common_test.go

```diff
@@ -1604,3 +1624,13 @@ func createArtifactFile(numBytes int64) (string, error) {
 	}
 	return outFile, nil
 }
+
+func modifyArtifactFile(path string, newNumBytes int64) error {
+	dir, last := filepath.Split(path)
+	session := podmanTest.Podman([]string{"run", "-v", fmt.Sprintf("%s:/artifacts:z", dir), ALPINE, "dd", "if=/dev/urandom", fmt.Sprintf("of=%s", filepath.Join("/artifacts", last)), "bs=1b", fmt.Sprintf("count=%d", newNumBytes)})
+	session.WaitWithDefaultTimeout()
+	if session.ExitCode() != 0 {
+		return errors.New("unable to generate artifact file")
+	}
+	return nil
+}
```

Comment:
This seems extremely wasteful to run container for this.
You can just create random byte sin go, i.e. `math/rand` and write them to the file.

Reply:
consider:

```
artifact1File, err := createArtifactFile(4192)
Expect(err).ToNot(HaveOccurred())
```

as used in https://github.com/containers/podman/blob/bc2763eac828173484b1b7104136d4b4ad364482/test/e2e/artifact_test.go#L22